### DISCUSSION TITLE
[ FF7 ] Error "oops! Something very bad happened." when i create a new game file

### DISCUSSION THREAD

**Discussion by Hollandkion** [julianxhokaxhiu/FFNx#189] [General]
This error occurs when i create a new save 
![Captura de Tela (77)](https://user-images.githubusercontent.com/71286024/93123117-fe999d80-f69d-11ea-8301-d18cd564d92c.png)
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5220231/FFNx.log)

**Comment by julianxhokaxhiu** [comment]
Hi,

Something in your setup has gone wrong. If you're using 7th it may be the upgrade process from an earlier version may have broken something. Please use the relative support channels for help.

Best regards :)
